hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/BlocksEndPoint.java

```diff
@@ -68,41 +68,66 @@ public BlocksEndPoint(ReconStorageContainerManagerFacade reconSCM) {
   }
 
   /**
-   * This API returns list of blocks grouped by container state
+   * This API returns the total count of blocks pending deletion and a list of
+   * blocks grouped by container state
    * (OPEN/CLOSING/CLOSED).
+   *
+   * Example of response:
    * {
-   *   "OPEN": [
-   *     {
-   *       "containerId": 100,
-   *       "localIDList": [
-   *         1,
-   *         2,
-   *         3,
-   *         4
-   *       ],
-   *       "localIDCount": 4,
-   *       "txID": 1
-   *     }
-   *   ]
+   *   "totalCount": 1000,
+   *   "containerStateBlockInfoListMap": {
```

Comment:
@ArafatKhan2198  thanks for working on this patch, We need total count and unreplicated and replicated size mapped to those blocks.  You should provide a new API, below API is only based on pagination.